### PR TITLE
Changed the modules to use x-vcloud-authorization headers.

### PR DESCRIPTION
**Pull request by AndreasMWalter**
As far as I have tested it, it should work now using the headers.
I haven't tested it on the old vCloud though, so you will need to wait for my confirmation there...

Ok I have tested the Connect-VcdOrganisation and the Get-VcdVM on the old API (Using Version 23.0) that worked without issues...

### CODE REVIEW

File: functions/Connect-VcdOrganisation.ps1 [hunk 1]
```diff
@@ -4,53 +4,54 @@ function Connect-VcdOrganisation {
...
-        #region Login and generate Session
-        # Add Authorization to header
+
+        #region Generate session headers
+        $headers = @{"Accept" = "application/*+xml;version=$APIVersion"}
         $headers += @{"Authorization" = "Basic $($EncodedAuthenticationStringBase64)"}
-        $SessionInformation = Invoke-RestMethod -Uri $loginurl -Headers $headers -Method POST -Session vCloudSession -ErrorAction Stop
-        if ($vCloudSession.Headers.Count -eq 0 ) {
+        Write-Verbose "Generated header"
+        [xml]$session
```
Comment (f-bader) [inline functions/Connect-VcdOrganisation.ps1]: Do you use this anywhere? If not please remove it. Best to check with PowerShell Script Analyzer

**Comment by f-bader** [comment]
Please review my code comment and follow up on it.

**Comment by AndreasMWalter** [comment]
Should be fine now, the Session variable was a remnant of the old code xD

**Comment by f-bader** [comment]
@AndreasMWalter Thank you, great first Pull Request

**Pull request merged**